plugins/org.eclipse.xtext.builder/src/org/eclipse/xtext/builder/nature/messages.properties

```diff
@@ -1,3 +1,3 @@
-NatureAddingEditorCallback_MessageDialog_Title=Add Xtext Nature
-NatureAddingEditorCallback_MessageDialog_Msg0=Do you want to add the Xtext nature to the project '
+NatureAddingEditorCallback_MessageDialog_Title=Configure Xtext
```

Comment:
I think this should use the same lingo than the configure action, i.e. "Convert to Xtext Project" and Miro's suggestion.